src/base64.h

```diff
@@ -52,8 +52,8 @@ extern const int8_t unbase64_table[256];
 
 
 template <typename TypeName>
-size_t base64_decode_slow(char* dst, size_t dstlen,
-                          const TypeName* src, size_t srclen) {
+size_t base64_decode_slow(char* const dst, const size_t dstlen,
+                          const TypeName* const src, const size_t srclen) {
```

Comment:
Neither of them is strictly necessary, but why not add it if we don't mutate the pointer? And that would be consistent with `base64_decode_slow()` (line 90) and `base64_decode()` (line 122).